### ISSUE TITLE
README feels a little outdated 

### ISSUE THREAD

**Issue by Artoria2e5** [rfbproto/rfbproto#53]
It feels like the README is not quite reflecting how this repo takes patches now (PR), nor is it up to date with how things are formatted. Maybe someone can find some time to update it?

**Comment by CendioOssman** [comment]
Yeah, I guess we haven't really enforced that. And might not want to either.

The initial hope was to get many VNC developers active on the mailing list, so we could have active discussions about the protocol. That hasn't actually materialised, unfortunately.
